ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -257,6 +271,80 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: { id: '8304', source: 'Athena', capture: false },
       response: Responses.aoe(),
     },
+    {
+      id: 'P12S Paradeigma Counter',
+      type: 'StartsUsing',
+      netRegex: { id: '82ED', source: 'Athena', capture: false },
+      run: (data) => data.paradeigmaCounter++,
+    },
+    {
+      id: 'P12S Paradeigma 1 Clones',
```

Comment:
I do wonder if everybody calls things para1 para2 para3 etc that the timeline should also number them? (This doesn't have to be done here, but just thinking out loud.)